### ISSUE TITLE
Create Hawaii Instance Files

### ISSUE THREAD

**Issue by jordanngtw** [PublicDataWorks/complaint-manager#226]
Should be done before or after #228, or at the very least in close concert if done concurrently

### Business Context:
We are building towards releasing a Hawaii version of Complaint Manager, and as we haven't fully excised instance-files from our process yet, we need to create an instance-files-hawaii-csoc (correctional system oversight commission) project (private GitHub repo in the PublicDataWorks project) following the pattern created with instance-files-noipm (I think the demo instance-files may be a little behind), so that we can build out Hawaii's use-case from there.  Don't worry about having accurate data just yet, but make sure to remove actual New Orleans names and emails

**As a** Complaint Manager
**I want** to be able to hold Hawaii specific info in a separate repo
**So that** I can be launched for Hawaii

### In Scope
1. Create instance-files-hawaii-csoc as a private GitHub repository inside PublicDataWorks
2. Copy over instance-files-noipm to instance-files-hawaii-csoc
3. Remove New Orleans names, emails, etc. from instance-files-hawaii-csoc

### Out of Scope
1. Putting in Hawaii's specific data (except PERSON_TYPEs if #228 is already done)
2. 
3.

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
4. Determine if documentation should be referenced or created.
5. Ensure to include all necessary mockups to this card.
6. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
7. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
8. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
9. Have we added edge cases as ACs that we're considered from a QA perspective
10. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**
